### ISSUE TITLE
import of already existing data

### ISSUE THREAD

**Issue by AKWB** [BU-ISCIII/iSkyLIMS#5]
Is it somehow possible to import already existing sequencing experiments to this LIMS?

**Comment by saramonzon** [comment]
Hi there!
Yes, it is possible! For MiSeq runs, it automatically finds new runs not present in the database and present in the samba share. For nextSeq data it is needed to import the sample_sheet and the experiment name first. This can be done:
- Manually using the web form, as they were new runs.
- Making a small python script, we can help you with this if you need it.

Thanks for posting!
Regards
Sara

**Comment by AKWB** [comment]
Hi Sara,

thanks a lot! I have some issues importing a sample sheet, which I actually created with the Illumina Experiment Manager (I even installed it just for iSkyLIMS). However there seems to be an issue with the parsing. Do you have somewhere a sample samplesheet.csv, where I could look what is the difference? Actually it is telling me that the Researcher XY is not in the database. But this "name" it is giving me, is the description field of the sample sheet. Thus, maybe there is an issue with the version of IEM ?

Greets

**Comment by saramonzon** [comment]
Hi!
You are right, we have to complete the wiki, it is lack of some important information.
About this you are totally right, we have to change it in the next release. I will try to explain myself.
 We have a script for bcl2fastq conversion and the fastqs generated are shared using also this script with the requesting user. In order to supply to the script the requesing user of each project in the sample sheet we use the description field. We add the user who owns that project/sample to the description field, and this is used for the script to create a samba share and send the files to the researcher, and it is used by iSkyLIMS to match each sample/project to the requesting user.

So in summary our code expects that you add the same used id in the description field and in the iSkyLIMS database!
I attach you a sample_sheet for testing. You just need to add the user1 and user2 to your database using the admin site in your installation, in User table-> add new user.

As I said before we will change this in the next release so it is not necessary to add the user in the description field. Also we will update the wiki with more information about this!

[sample_sheet_test.txt](https://github.com/BU-ISCIII/iSkyLIMS/files/2856484/sample_sheet_test.txt)

Thanks a lot for your input!

**Comment by luissian** [comment]
Code is already updated with this issue.
From now on, it is not needed to include in the sample sheet the user name under Description column.
But as a mandatory information it has to be provided in the form when creating the new NextSeq run.
JavaScript function is implemented to check if a valid user name is typed. It means that accepted user names must be defined in advance using the admin feature to create a new user.
Thanks again for your input

**Comment by AKWB** [comment]
back to this issue :-). We have aMiniSeq - so BaseSpace etc is not an option for us. Would you help me to get run data imported to iSkyLIMS?
